app/views/rescue_codes/new.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('titles.rescue_codes')
+
+h1.heading = t('headings.rescue_codes')
+p
+  | Below are a set of rescue codes that we have generated for you.
+    You can use these codes to access your account in case you loose
+    access to your phone. Please download these codes and keep them
+    in a safe place.
```

Comment:
I agree that we only need to provide one recovery code at a time to each user and, naturally, I like the  instructions. We'll need to generate a new recovery code every time to recover their account though. 

The download feature is actually quite practical for users and the whole reason @sbc100 implemented it this way was because a person on the team recently had to recover their GitHub account and the backup codes were still sitting in her Downloads folder.